### ISSUE TITLE
missing gulp task example for JavaScript

### ISSUE THREAD

**Issue by schilke** [chrismcintosh/bolt-pro#10]
I'd love to see a (roughly working?) example here, like sass, browsersync and watch. 
As a beginner with gulp it's a bit hard to set this up from scratch (because of the modular structure  - which is surely a good thing but differs from the original Foundation setup).

**Comment by chrismcintosh** [comment]
Juergen,

I'm not sure I completely understand your request. Are you asking about
getting these tools working in a non bolt-pro project?

If you are asking for a walkthrough of how to get started with gulp with
Bolt Pro check out this youtube video I put together of me pulling the repo
down and starting a project and getting gulp running
https://www.youtube.com/watch?v=dW0ZWKfIWEc [7 minutes]

I'm happy to help I just need a better idea of what you need help with.

**Comment by schilke** [comment]
Chris,

thanks for the video(s) - awesome :+1:  but I've already watched all of them :)

And no, I don't want to work on a non bolt-pro project - quite the opposite applies but I'd like to start with everything needed in place and while almost everything else is prepared perfectly (setup/install, the (S)CSS workflow, browsersync, watch etc.) there's no JS compiling at all.
I see there's something in `/assets/gulp/config.js` but that's all about JS I found.

I'd love to see the (JS) dependencies included as well as a simple example setup like for the other gulp tasks. Besides Zurb itself similar projects all do this somehow (you probably know), e.g. 
[FoundationPress](https://github.com/olefredrik/FoundationPress)
[genesis-foundation](https://github.com/secretstache/genesis-foundation)
[JointsWP](https://github.com/JeremyEnglert/JointsWP)
or [heisenberg](https://github.com/ZeekInteractive/heisenberg) (On a side note I like that they also dropped Bower completely and do the Foundation 6 install via npm).

No worries as I'll find a way around this - but this would definitely look more complete to me. I hope this makes sense.
